src/yb/yql/cql/ql/util/cql_message.cc

```diff
@@ -390,6 +403,16 @@ bool CQLRequest::ParseRequest(
   return true;
 }
 
+int64_t CQLRequest::ParseRpcQueueLimit(const Slice& mesg) {
+  Flags flags = LoadByte<Flags>(mesg, kHeaderPosFlags);
+  if (!(flags & kMetadataFlag)) {
+    return 0;
```

Comment:
Looks like it would be more clear to return `std::numeric_limits<int64_t>::max()` in this case.